Source/Core/DolphinQt/RenderWidget.cpp

```diff
@@ -254,6 +254,10 @@ void RenderWidget::SetCursorLocked(bool locked, bool follow_aspect_ratio)
   if (locked)
   {
 #ifdef _WIN32
+    // This will prevent the mouse from interacting with the task bar
+    // while in windowed / borderless fullscreen.
+    SetWindowPos((HWND) winId(), HWND_TOPMOST, 0, 0, 0, 0, SWP_NOMOVE | SWP_NOSIZE | SWP_NOACTIVATE);
```

Comment:
Have you tested this with "Render To Main Window"? I don't know what happens if you set HWND_TOPMOST to a window handle that is part of another window.

Reply:
Interestingly, using Dolphin's built in `Keep Window on Top` itself doesn't seem to work with "Render To Main Window" in the first place. As such, this implementation doesn't work with that option either. The fix should be simple though.

Reply:
> 
> 
> Interestingly, using Dolphin's built in `Keep Window on Top` itself doesn't seem to work with "Render To Main Window" in the first place. As such, this implementation doesn't work with that option either. The fix should be simple though.

That's a good idea for a follow up pr.

Reply:
Not really, this fix should work with all dolphin settings out of the box.

Reply:
I'm a little confused, but I'll take your word for it 😁